### DISCUSSION TITLE
yiwen-blog-website/docs/native/ReactNative/fabric

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#171] [Announcements]
# yiwen-blog-website/docs/native/ReactNative/fabric

class组件和function组件在状态变量中的区别

https://hao-yiwen.github.io/yiwen-blog-website/docs/native/ReactNative/fabric

**Comment by Hao-yiwen** [comment]
旧项目适配新架构发现很多写法不兼容，不看好jsi前途。虽然描述很厉害，但是实际开发体验并不友好。jsi还需要进一步的完善和优化。

**Comment by Hao-yiwen** [comment]
https://github.com/reactwg/react-native-new-architecture#guides
